apps/test-netzer-encoder/include/NetzerEncoder.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "mbed.h"
+
+class NetzerEncoder {
+
+	public:
+		NetzerEncoder(int frequency);
```

Comment:
users shouldnt have to pass in the frequency. its an internal detail. make the frequency a private static constexpr in this class